src/eXpansion/Bundle/Menu/Model/Menu/AbstractItem.php

```diff
@@ -84,6 +84,11 @@ public function permission()
         return $this->permission;
     }
 
+    public function addChild($class, $id, $label, $permission, $options)
```

Comment:
Why? And it does nothing I am lost ?

Reply:
it's to normalize the interfaces for IDE syntax checking...

Reply:
No, 

It's not okay to add empty methods just for IDE checkings. Need to be removed.